libs/features/mas/src/merch-card.js

```diff
@@ -376,7 +376,7 @@ export class MerchCard extends LitElement {
         const timeoutPromise = new Promise((resolve) =>
             setTimeout(() => resolve(false), MERCH_CARD_LOAD_TIMEOUT),
         );
-        const success = await Promise.race([successPromise, timeoutPromise, this.aemFragment?.updateComplete]);
+        const success = await Promise.race([successPromise, timeoutPromise]);
```

Comment:
@bozojovicic didn't you add `this.aemFragment?.updateComplete`? 
I think we need to keep it.

Reply:
not sure we can keep it if we want merch-card to fire an error when price/cta failed - and thats our contract for ccd at the moment

Reply:
this basically means we ignore successPromise

Reply:
@bozojovicic noticed in his autoblock PR, that when you call `merchCard.checkReady()` too early, it does not work (gives false positive) because you first need to wait for the fragment to be loaded.
if the fragment is not resolved, we cannot know which offers are to be loaded anyway.